src/utils/types/gameState.d.ts

```diff
@@ -5,6 +5,12 @@
  * and run json-schema-to-typescript to regenerate this file.
  */
 
+export type RoomState = ExampleRoom1;
+
 export interface GameState {
-  currentRoomId: null;
+  roomState: RoomState;
+}
+export interface ExampleRoom1 {
+  type: 'exampleRoom1';
+  playerState: 'AtEntrance';
```

Comment:
Whenever I run `npm run compile:gameStateSchema` it changes this to:
```
export interface ExampleRoom1 {
  type: "exampleRoom1";
  playerState: "AtEntrance";
}
```
Is there a setting for quotes?

Side note: Should this file be commited?

Reply:
I was wondering both of those things. I think we can resolve all of that outside this PR